lib/transports.js

```diff
@@ -31,14 +34,45 @@ HTTPTransport.prototype.send = function(client, message, headers, eventId, cb) {
     ca: client.ca,
     agent: this.agent
   };
+
+  // set path apprpriately when using http endpoint + proxy, set proxy headers appropriately when using https endpoint + proxy
+  if (this.options.hasOwnProperty('proxyHost')) {
+    if (client.dsn.protocol === 'http') {
+      this.options.path =
+        'http://' +
+        client.dsn.host +
+        ':' +
+        client.dsn.port +
+        client.dsn.path +
+        'api/' +
+        client.dsn.project_id +
+        '/store/';
+      delete options.hostname; // only 'host' should be set when using proxy
```

Comment:
slightly better comment here for what will happen if you send hostname or a link to a resource that justifies why?